src/lib.rs

```diff
@@ -394,8 +401,21 @@ pub fn set_encounter(
 #[cfg(feature = "wasm")]
 #[wasm_bindgen(js_name = "setLoggingLevel")]
 pub fn set_logging_level(_level: usize) -> Result<(), JsValue> {
+    let level = match _level {
+        0 => tracing::Level::TRACE,
+        1 => tracing::Level::DEBUG,
+        2 => tracing::Level::INFO,
+        3 => tracing::Level::WARN,
+        4 => tracing::Level::ERROR,
```

Comment:
Note that this the _opposite_ of what's happening inside of the (since-deleted) `logging.rs`—both `log` and `tracing` treat the most verbose level as the lowest usize, but `logging` did the opposite (0 is `ERROR`).

I put up this PR as-is since I'm not sure what the usage of this is over on d2foundry (and since `log`/`tracing` have one more logging level than D2_Calculation_API, starting at 0 would result in an off-by-one-error semantic error _anyways_).

In any case, I'm happy to change this section however you'd like me to.